
An ugly patch for an ugly lvalue cast - for gcc-3.4.

  - Rene Rebe <rene@rocklinux.org>

--- ./src/ui/ncurses/uitagbrowser.cc.vanilla	2004-07-05 14:13:59.000000000 +0200
+++ ./src/ui/ncurses/uitagbrowser.cc	2004-07-05 14:08:51.000000000 +0200
@@ -78,7 +78,8 @@
     cm.setwindow(textwindow(w.x1, w.y1, w.x1+20, 0, uiconf.getcolor(cp_menufr)));
 
     for(motortagbrowser::viewscope scope = motortagbrowser::File;
-    scope != motortagbrowser::viewscope_size; (int) scope += 1) {
+    scope != motortagbrowser::viewscope_size;
+    scope = (motortagbrowser::viewscope) ((int)scope + 1) ) {
 	cm.additem(0, scope, (string) " " + getscopename(scope));
 	if(scope == tagbrowser.getscope())
 	    cm.setpos(cm.getcount()-1);
